tests/e2e/tests/pilot/routing_test.go

```diff
@@ -193,6 +193,30 @@ func TestRoutes(t *testing.T) {
 			expectedCount: map[string]int{"v1": 100, "v2": 0},
 			operation:     "",
 		},
+		{
+			testName:      "a->c[v2=100]_tcp_single",
+			description:   "routing tcp traffic from a to single dest c-v2",
+			config:        "virtualservice-route-tcp-weighted.yaml",
+			scheme:        "http",
+			src:           "a",
+			dst:           "c:90",
+			headerKey:     "",
+			headerVal:     "",
+			expectedCount: map[string]int{"v1": 0, "v2": 100},
+			operation:     "",
+		},
```

Comment:
remove this test. We have plenty of TCP routing tests in tcp_test.go or tls_test.go I think.

Reply:
This is slightly different because the weight isn't specified [here](https://github.com/istio/istio/pull/9112/files/209a6170fbcc0a69f4be711077c20faf728f329d#diff-80b76ed56247507b57aaa7c2077fc330R15).